tests/regression/test_suite/find_transactions_hash_by_tag.py

```diff
@@ -0,0 +1,92 @@
+from common import *
+import json
+import unittest
+import time
+import logging
+
+
+class FindTransactionsHashByTag(unittest.TestCase):
+
+    # Normal 27 trytes tag (pass)
+    @test_logger
+    def test_27_trytes(self):
+        res = API(f"/tag/{self.query_string[0]}/hashes", get_data="")
+        self._verify_pass(res)
+
+    # Normal 27 trytes tag with trailing slash (pass)
+    @test_logger
+    def test_with_trailing_slash(self):
```

Comment:
What's the different between `test_27_trytes` and `test_with_trailing_slash`?

Reply:
`test_with_trailing_slash` has a slash at the end of the url